.travis.yml

```diff
@@ -17,14 +17,17 @@ env:
   - THREADSAFE=true DB=postgresql
   - THREADSAFE=false DB=postgresql
 gemfile:
+  - gemfiles/Gemfile.rails-5.0-stable
   - gemfiles/Gemfile.rails-4.2-stable
   - gemfiles/Gemfile.rails-4.1-stable
   - gemfiles/Gemfile.rails-4.0-stable
   - gemfiles/Gemfile.rails-3.2-stable
   - Gemfile
 matrix:
   allow_failures:
-    - gemfile: Gemfile # Not supporting rails 5 at the moment
+    - rvm: 2.0.0
+    - rvm: 2.1.0
```

Comment:
Argh, ok, let's keep this way then, but can you amend the commit and drop in the body explaining the reason behind the change?